app/src/main/java/org/mifos/mobile/ui/fragments/AddGuarantorFragment.kt

```diff
@@ -171,10 +164,10 @@ class AddGuarantorFragment : BaseFragment(), AddGuarantorView {
     override fun showGuarantorUpdation(template: GuarantorTemplatePayload?) {
         this.template = template
         setUpSpinner()
-        tilFirstName?.editText?.setText(payload?.firstname)
-        tilLastName?.editText?.setText(payload?.lastname)
-        tilOfficeName?.editText?.setText(payload?.officeName)
-        (guarantorTypeField?.editText as?  MaterialAutoCompleteTextView)?.setText(payload?.guarantorType?.value!!)
+        binding.tilFirstName.editText?.setText(payload?.firstname)
+        binding.tilLastName.editText?.setText(payload?.lastname)
+        binding.tilOfficeName.editText?.setText(payload?.officeName)
+        (binding.guarantorTypeField.editText as?  MaterialAutoCompleteTextView)?.setText(payload?.guarantorType?.value!!)
 
     }
 
```

Comment:
It seems findPreviouslySelectedGurantorType() is not being used anywhere, can be removed after a thorough usage check.

Reply:
done @gururani-abhishek  , I have also used `with(binding)`